examples/source_mapping/app.py

```diff
@@ -0,0 +1,25 @@
+# Import pyteal _after_ setting feature gate option
+from pyteal import Assert, Expr, Int, Seq, abi
```

Comment:
can we change these to `import pyteal as pt` and `import beaker` to follow other examples?
side note, I think I missed some in `simple` when doing the homogenisation

Reply:
went halfway on this, aliasing pyteal to pt. 

Planning to do another docs pr and I'll do the beaker one there since itll change line numbers on the examples in docs and that requires some manual work